vcd/resource_vcd_vapp_firewall_rules.go

```diff
@@ -49,6 +49,12 @@ func resourceVcdVappFirewallRules() *schema.Resource {
 				ForceNew:    true,
 				Description: "vApp network identifier",
 			},
+			"enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Enable or disable firewall service.",
```

Comment:
Should state the default in description too.

Reply:
added